### ISSUE TITLE
Is BAAI/bge-small-en-v1.5 supported?

### ISSUE THREAD

**Issue by dsp05** [qdrant/fastembed#11]
It looks like `fast-bge-small-en-v1.5.tar.gz` exists on GCS but is not complete.

```
    259 tokenizer_path = model_dir / "tokenizer.json"
    260 if not tokenizer_path.exists():
--> 261     raise ValueError(f"Could not find tokenizer.json in {model_dir}")
    262 model_path = model_dir / "model_optimized.onnx"
    263 if not model_path.exists():
```

**Comment by NirantK** [comment]
That's work in progress! Will support with the 0.0.6 launch and change the default to that model as well. Expect this to be around 2023-10-03

**Comment by NirantK** [comment]
Running behind schedule on this because of bugs in the 0.0.5 alpha releases. Still intend to have this supported with 0.0.6!

**Comment by NirantK** [comment]
This is now supported in `main` and will be released with 0.0.6 if you're looking for a pinned verison

**Comment by NirantK** [comment]
Supported in #25
